backend/src/schema/resolvers/registration.js

```diff
@@ -13,7 +13,12 @@ export default {
       args.nonce = generateNonce()
       args.email = normalizeEmail(args.email)
       let emailAddress = await existingEmailAddress({ args, context })
-      if (emailAddress) return emailAddress
+      /*
+      if (emailAddress.user) {
+        // what to do?
+      }
+      */
+      if (emailAddress.alreadyExistingEmail) return emailAddress.alreadyExistingEmail
```

Comment:
This is just for admin invitation, as far I can see.

May be show the admin somehow (modal dialog or alse) that the e-mail is already registered?